src/ProjectNeon/Assets/Features/Effects/Events/AttackPerformed.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+public sealed class AttackPerformed
+{
+    public Attack Attack { get; }
+
+    public AttackPerformed(Attack attack, Member attacker, Target target)
```

Comment:
The attacker and target are not being put in the message.

Reply:
@SilasReinagel we already have reference to these in `Attack`